web_widget_ckeditor/static/src/js/field_ckeditor.js

```diff
@@ -0,0 +1,341 @@
+                const value = this._textToHtml(this.value);
+                this.$target = $("<textarea>").val(value).hide();
+                this.$target.appendTo(this.$el);
+                return this._createCKEditorIntance();
+            },
+            /**
+             * @override
+             */
+            _renderReadonly: function () {
+                const value = this._textToHtml(this.value);
+                this.$el.empty();
+                this.$content = $('<div class="o_readonly"/>').html(value);
+                this.$content.appendTo(this.$el);
+            },
+            /**
+             * This function is similar to the one found in core's web_editor.FieldHtml.
+             *
+             * @private
+             * @param {String} text
+             * @returns {String} the text converted to html
+             */
+            _textToHtml: function (text) {
+                let value = text || "";
+                try {
+                    // Crashes if text isn't html
+                    $(text)[0].innerHTML; // eslint-disable-line
+                } catch (e) {
+                    if (value.match(/^\s*$/)) {
+                        value = "<p><br/></p>";
+                    } else {
+                        value =
+                            "<p>" +
+                            value.split(/<br\/?>/).join("<br/></p><p>") +
+                            "</p>";
+                        value = value
+                            .replace(/<p><\/p>/g, "")
+                            .replace("<p><p>", "<p>")
+                            .replace("<p><p ", "<p ")
+                            .replace("</p></p>", "</p>");
+                    }
+                }
+                return value;
+            },
+
+            // --------------------------------------------------------------------------
+            // Handler
+            // --------------------------------------------------------------------------
+
+            /**
+             * Method called when the CKEditor instance is loaded.
+             *
+             * @private
+             */
+            _onLoadCKEditor: function () {
+                const $button = this._renderTranslateButton();
+                $button.css({
+                    "font-size": "15px",
+                    position: "absolute",
+                    right: "+5px",
+                    top: "+5px",
+                });
+                this.$el.append($button);
+            },
+            /**
+             * Method called when ckeditor triggers a change.
+             *
+             * @private
+             * @param {OdooEvent} ev
+             */
+            _onChange: function () {
+                this._doDebouncedAction.apply(this, arguments);
+            },
+        }
+    );
+
+    field_registry.add("ckeditor", FieldHtmlCKEditor);
+
+    // Replace core's html widget with ckeditor, depending on the ir.config_parameter
```

Comment:
IMO this would be easier by loading another asset view on demand: use the flag `active` to disable it.

Reply:
Thanks!! Indeed I like this approach. I've just pushed the changes